packages/11ty/_plugins/shortcodes/figure.js

```diff
@@ -46,16 +46,12 @@ module.exports = function (eleventyConfig, globalData) {
           return figureplaceholder(figure)
         case mediaType === 'youtube':
           return figureyoutube(figure)
-          break
         case mediaType === 'vimeo':
-          break
+          return 'UNIMPLEMENTED'
         case mediaType === 'soundcloud':
           return figuresoundcloud(figure)
-        case mediaType === 'website':
-          break
         case mediaType === 'table':
-          return figuretable(figure)
-          break
+          return await figuretable(figure)
```

Comment:
this table stuff must have gotten merged in while I was cleaning up branches, my bad